src/wasm/wasm-type.cpp

```diff
@@ -411,6 +411,13 @@ bool Type::isStruct() const { return isRef() && getHeapType().isStruct(); }
 
 bool Type::isArray() const { return isRef() && getHeapType().isArray(); }
 
+bool Type::isDefaultable() const {
+  // A variable can get a default value if its type is concrete (unreachable
+  // and none have no values, hence no default), and if it's a reference, it
+  // must be nullable.
+  return isConcrete() && (!isRef() || isNullable());
```

Comment:
Do RTTs have default values?

Reply:
I think so, and binaryen gives them the natural default value (`rtt.canon`). I admit I can't find an explanation of that on the GC docs, though. I opened https://github.com/WebAssembly/gc/issues/175